packages/multi-select-combo-box/theme/lumo/vaadin-multi-select-combo-box.js

```diff
@@ -3,7 +3,6 @@
  * Copyright (c) 2021 - 2023 Vaadin Ltd.
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
-import '@vaadin/overlay/theme/lumo/vaadin-overlay.js';
```

Comment:
This theme import was added to overlay based components in #3888 but it's no longer needed: from now on, `vaadin-overlay` doesn't get defined when importing `vaadin-multi-select-combo-box`.